fendermint/app/src/main.rs

```diff
@@ -30,6 +32,58 @@ async fn main() {
     }
 }
 
+fn create_log(level: tracing::Level, log_dir: &Path) -> anyhow::Result<()> {
+    let log_folder = expand_tilde(log_dir)
+        .to_str()
+        .ok_or_else(|| anyhow!("cannot parse log folder"))?
+        .to_string();
+    std::fs::create_dir_all(&log_folder).context("cannot create log folder")?;
+
+    let console_appender = config::Appender::Console;
+    let topdown_appender = config::Appender::RollingFile {
+        path: format!("{log_folder}/topdown.log"),
+        policy: Policy {
+            maximum_file_size: "10mb".to_string(),
+            // we keep the last 5 log files, older files will be deleted
+            max_size_roll_backups: 5,
+            pattern: None,
+        },
+    };
+
+    // debug level logger
+    let topdown_logger = config::Logger {
+        level: config::LevelFilter::DEBUG,
+        appenders: literally::hset! { AppenderId::from("topdown") },
+        format: Format::default(),
+    };
+
+    // default logging output info log to console
+    let default = config::Logger {
+        level: config::LevelFilter::from_str(level.as_str())?,
+        appenders: literally::hset! { AppenderId::from("console") },
+        format: Format::default(),
+    };
+
+    let config = Config {
+        default,
+        loggers: literally::hmap! {
+            Target::from("fendermint_vm_topdown") => topdown_logger.clone(),
+            Target::from("fendermint_vm_interpreter::chain") => topdown_logger,
+        },
+        appenders: literally::hmap! {
+            AppenderId::from("console") => console_appender,
+            AppenderId::from("topdown") => topdown_appender
```

Comment:
Note that this is in `main.rs` which is also the entry point for the `ethapi` as well as the CLI. Those won't have these emitters, but I can imagine that they would overwrite logs of each other when executed on the host directly, without a container. 

Maybe it would be better if the log directory wasn't specified by default, so that the operator has to explicitly set it, which would make any clashes obvious.